x-pack/plugins/fleet/server/types/models/output.ts

```diff
@@ -115,12 +115,30 @@ export const ElasticSearchSchema = {
   ...BaseSchema,
   type: schema.literal(outputType.Elasticsearch),
   hosts: schema.arrayOf(schema.uri({ scheme: ['http', 'https'] }), { minSize: 1 }),
+  preset: schema.maybe(
```

Comment:
should we have a default value for `preset`?

Reply:
Added a check to make sure we don't compare an existing output where `preset` has been defaulted to a preconfigured one where it hasn't been specified.